pkg/section/module.go

```diff
@@ -0,0 +1,75 @@
+package section
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+
+	"github.com/daixiang0/gci/pkg/parse"
+	"github.com/daixiang0/gci/pkg/specificity"
+)
+
+const ModuleType = "module"
+
+type Module struct {
+	ModulePaths []string
+}
+
+func (m *Module) MatchSpecificity(spec *parse.GciImports) specificity.MatchSpecificity {
+	for _, modPath := range m.ModulePaths {
+		if strings.HasPrefix(spec.Path, modPath) {
+			return specificity.Module{}
+		}
+	}
+
+	return specificity.MisMatch{}
+}
+
+func (m *Module) String() string {
+	return ModuleType
+}
+
+func (m *Module) Type() string {
+	return ModuleType
+}
+
+// Configure configures the module section by finding the module
+// for the current path
+func (m *Module) Configure() error {
+	modPaths, err := findModules()
+	if err != nil {
+		return err
+	}
+	m.ModulePaths = modPaths
+	return nil
+}
+
+func findModules() ([]string, error) {
+	packages, err := packages.Load(
+		// find the package in the current dir and load its module
+		&packages.Config{Mode: packages.NeedModule},
+		".",
+	)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load local modules: %v", err)
+	}
+
+	uniqueModules := make(map[string]struct{})
+
+	for _, pkg := range packages {
+		if len(pkg.Errors) != 0 {
+			return nil, fmt.Errorf("error while listing modules: %v", pkg.Errors[0])
```

Comment:
Errors or Error[0]?

Reply:
> Errors or Error[0]?

:+1: I'm not sure how common multiple errors are, but `Errors` seem safer (it should communicate more information)

https://github.com/daixiang0/gci/pull/179/files#diff-f775f605b71efdb8d7d023d34bc715f03e8fa295b8f96aea2743698b24f9f285R64